examples/mf6/hondsrug.py

```diff
@@ -31,8 +31,7 @@
 
 # %%
 # Before starting to create the input data, we will create the groundwater
-# model variable (gwf_model) using `imod.mf6.GroundwaterFlowModel
-# <https://deltares.gitlab.io/imod/imod-python/api/mf6.html?highlight=groundwater%20flow%20model#imod.mf6.GroundwaterFlowModel>`_.
+# :doc:`/api/generated/mf6/imod.mf6.GroundwaterFlowModel`.
```

Comment:
Have you tested locally if the ``:doc:`` reference is properly rendered in the html pages?

Reply:
Yes I tested this, Everything is rendered except for the references in the [example_models.py](https://github.com/Deltares/imod-python/pull/701/files#diff-560f1c803908404450401380d544d156dcb3c0dbf555a5ba9ff8f4b305952560). That because the example_models.py file is included in 1 major code block

Reply:
Great, thanks! I don't think the example_models.py script is rendered in the docs anyway.